lib/transforms/subsetFonts.js

```diff
@@ -419,25 +480,31 @@ module.exports = ({
 
             assetGraph.eachAssetPreOrder(htmlAsset, traversalRelationQuery, asset => {
                 if (asset.type === 'Css' && asset.isLoaded) {
+                    const seenNodes = [];
```

Comment:
Use a `Set` to avoid the O(n)-ness of `Array#includes`?

Reply:
Done